backend/db/sqliteschema/migrations.go

```diff
@@ -219,4 +219,66 @@ var migrations = []string{
 		-- The balance in satoshis
 		balance INTEGER DEFAULT 0
 	);`,
+
+	// Stores sites that user has manually added
+	`CREATE TABLE sites (
+		-- Site unique identification. The hostname of the site with protocol https://example.com
+		hostname TEXT PRIMARY KEY,
+		-- The role we play in the site ROLE_UNSPECIFIED = 0 | OWNER = 1 | EDITOR = 2
+		role INT NOT NULL DEFAULT 0,
+		-- Address of the LND node backing up this wallet. In case lndhub, this will be the 
+		-- P2P addresses to connect to that site in the format of multiaddresses. Space separated.
```

Comment:
I'm confised with what lndhub and wallet stuff has to do with sites here. Is this a copy-paste maybe?